source/text/Format.cpp

```diff
@@ -372,6 +373,75 @@ string Format::LowerCase(const string &str)
 
 
 
+// Convert the given non-empty string (noun) to its plural form.
+// Always returns a non-empty string.
+// The actual result might involve some guesswork, and is not guaranteed to be grammatically correct.
+string Format::Plural(const string &noun)
+{
+	if(noun.empty())
+		return noun;
+	static const set<string> exceptionEs={"buffalo", "domino", "echo", "embargo", "hero",
```

Comment:
`exceptionEs`?
Also, maybe give each element its own line?
Also, spacing of the assignment operator. ~~Why isn't the check catching this?~~